src/components/authorize/claims-config.ts

```diff
@@ -9,30 +9,43 @@ export function getKnownClaims(): {
   };
 }
 
-// Single source of truth for claims type and claims object keys
+export type Claims = {
+  iss: string;
+  aud: string;
+  exp: number;
+  iat: number;
+  nbf: number;
+  jti: string;
+  client_name: string;
+  cookie_consent_shared: boolean;
+  consent_required: boolean;
+  is_one_login_service: boolean;
+  service_type: string;
+  govuk_signin_journey_id: string;
+  confidence: string;
+  state: string;
+  client_id: string;
+  scope: string;
```

Comment:
Where is this scope object coming from? I might have misunderstood (probably) but I thought you were getting rid of it from the claims JWT?: https://github.com/alphagov/di-authentication-api/pull/3313/files#diff-e0d0ad45a3807a8326404be3bcf0d979b5fd01235b7a751394012ca6a4c708c7L387

Reply:
Good point, will remove, we were thinking to keep it to avoid breaking current behaviour, but not realising the auth controller is not in current use